docs/ref/contrib/admin/filters.txt

```diff
@@ -0,0 +1,188 @@
+.. _modeladmin-list-filters:
+
+=======================
+ModelAdmin List Filters
+=======================
+
+.. currentmodule:: django.contrib.admin
+
+Model admin classes can define list filters that appear in the right sidebar of
+the change list page of the admin, as illustrated in the following screenshot:
+
+.. image:: _images/list_filter.png
+
+To activate per-field filtering, set :attr:`ModelAdmin.list_filter` to a list
+or tuple of elements, where each element is one of the following types:
+
+#. A field name
+#. A subclass of ``django.contrib.admin.SimpleListFilter``
+#. A tuple containing a field name and a subclass of
+   ``django.contrib.admin.FieldListFilter``
+
+See the examples below for discussion of each of these options for defining
+``list_filter``.
+
+Using a field name
+==================
+
+The simplest option is to specify the required field names.
+
+Each specified field should be either a ``BooleanField``, ``CharField``,
+``DateField``, ``DateTimeField``, ``IntegerField``, ``ForeignKey`` or
+``ManyToManyField``, for example::
+
+    class PersonAdmin(admin.ModelAdmin):
+        list_filter = ('is_staff', 'company')
+
+Field names in ``list_filter`` can also span relations
+using the ``__`` lookup, for example::
+
+    class PersonAdmin(admin.UserAdmin):
+        list_filter = ('company__name',)
+
+
+Using a ``SimpleListFilter``
+============================
+
+For custom filtering, you can define your own list filter by inheriting
+``django.contrib.admin.SimpleListFilter`` class for which you need to provide
+the ``title`` and ``parameter_name`` attributes to and override the ``lookups``
```

Comment:
I know it's the same wording that's already there, but `... attributes to, and override the ...` (addition of comma) would separate the actions better IMHO.

Reply:
Since you've added "for" before "which", the "to" after "attributes" is no longer needed. I agree a comma adds clarity.

Personally, I'd take it a step further and split it into two separate sentences; something along the lines of:

> For custom filtering, you can define your own list filter by inheriting from ``django.contrib.admin.SimpleListFilter``. You will need to provide the ``title`` and ``parameter_name`` attributes, and override the ``lookups`` and ``queryset`` methods, e.g.::

This is very much a question of individual taste though.

Reply:
OK, thanks both. Adjusted. 👍